agent/package.json

```diff
@@ -3,12 +3,13 @@
   "packageManager": "pnpm@9.15.3+sha512.1f79bc245a66eb0b07c5d4d83131240774642caaa86ef7d0434ab47c0d16f66b04e21e0c086eb61e62c77efc4d7f7ec071afad3796af64892fae66509173893a",
   "dependencies": {
     "@ai-sdk/anthropic": "^1.0.2",
-    "@animo-id/mdoc": "^0.2.39",
+    "@animo-id/mdoc": "^0.4.0",
     "@credo-ts/askar": "npm:@animo-id/credo-ts-askar@0.5.14-alpha-20250204074333",
     "@credo-ts/core": "npm:@animo-id/credo-ts-core@0.5.14-alpha-20250204074333",
     "@credo-ts/node": "npm:@animo-id/credo-ts-node@0.5.14-alpha-20250204074333",
     "@credo-ts/openid4vc": "npm:@animo-id/credo-ts-openid4vc@0.5.14-alpha-20250204074333",
     "@hyperledger/aries-askar-nodejs": "^0.2.3",
+    "@sphereon/pex": "5.0.0-unstable.28",
```

Comment:
launching the agent complaint that this missed, not sure if this is the correct fix.